src/components/styles/NavBar/NavBar.js

```diff
@@ -327,7 +327,7 @@ color: ${props => props.theme.global.textColor};
 `;
 
 const Brand = styled.div`
-  align-items: stretch;
+  align-items: center;
```

Comment:
Honestly, I'm a tad curious at the difference b/w these two. I should play around with them...

Reply:
Refreshing myself here https://css-tricks.com/snippets/css/a-guide-to-flexbox/ It kinda just seems like the stretch is the same as center, the only difference being that stretch fills up the container it's in. 🤷‍♂

Reply:
I'd keep this as is imo, it's from bulimia, and I think on "near-tablets" having the nav spread out across the space is easier to navigate with.

Reply:
Fixed this with recent commit. There was an issue with the nav not accepting the props for height